wagtail_wordpress_import/test/tests/test_wordpress_item.py

```diff
@@ -327,4 +326,30 @@ def test_items_dict_4(self):
         # self.items_dict[4] = has no wp:post_meta items
         wordpress_item = WordpressItem(self.items_dict[4], self.logger)
         with self.assertRaises(KeyError):
-            post_meta = wordpress_item.clean_wp_post_meta()["wp:postmeta"]
+            wordpress_item.clean_wp_post_meta()["wp:postmeta"]
+
+    def test_items_dict_1_excluded_keys(self):
+        wordpress_item = WordpressItem(self.items_dict[1], self.logger)
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["wp:postmeta"]
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["wp_post_meta"]
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["content:encoded"]
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["dc:creator"]
+        with self.assertRaises(KeyError):
+            wordpress_item.clean_wp_post_meta()["wp:post_id"]
+
+    def test_items_dict_1_included_keys(self):
+        wordpress_item = WordpressItem(self.items_dict[1], self.logger)
+        self.assertTrue("title" in wordpress_item.clean_wp_post_meta())
```

Comment:
**Suggestion**: Ditto, except here the benefit is cleaner code to read, and faster tests (due to calling a function only once).